src/components/country-challenges-chart/country-challenges-modal-additional/country-challenges-modal-additional-component.jsx

```diff
@@ -26,25 +26,6 @@ const CountryChallengesModalAdditional = () => {
       <div className={styles.countriesLegend}>
         {CONTINENTS.map((continent) => renderContinent(continent))}
       </div>
-      <div className={styles.sizesLegend}>
```

Comment:
There might be some styles code that we need to remove too

Reply:
:+1: